src/platform_specific/affinity.unix.cc

```diff
@@ -13,6 +12,7 @@ namespace detail {
 	uint32_t affinity_cores_available() {
 		cpu_set_t available_cores;
 		const auto affinity_error = pthread_getaffinity_np(pthread_self(), sizeof(cpu_set_t), &available_cores);
+		(void)affinity_error;
```

Comment:
I would like a `[[maybe_unused]]` on the declaration itself better (same below).